source/js/components/join/join.scss

```diff
@@ -2,27 +2,68 @@
   background: inherit;
 
   .has-danger {
-    color: $red;
+    color: $dark-red;
+
+    .form-control {
+      border: 2px solid $dark-red;
+    }
+
+    .glyph-container {
+      position: absolute;
+      top: 0;
+      bottom: 0;
+      right: 10px;
+      height: 13px;
+      width: 13px;
+      margin: auto;
+    }
+
+    .form-error-glyph:nth-child(2) {
+      @media (min-width: $bp-sm) {
+        margin-left: 30px;
+      }
+    }
   }
 
   .form-control {
     font-family: "Nunito Sans";
     font-weight: 400;
+    color: $black;
   }
 
   .fields-wrapper {
     flex: 1 1 auto;
   }
 
   @at-root .dark-theme & {
-    form {
-      input {
-        border: none;
-      }
-    }
-
     .body-small.form-control-feedback {
```

Comment:
Do we need to target `.body-small` here? Using just `.form-control-feedback` as selector should be enough.